vitest.config.ts

```diff
@@ -6,7 +6,7 @@ export default defineConfig({
   test: {
     environment: 'jsdom',
     setupFiles: './vitest.setup.ts',
-    exclude: ['**/__tests__/**/*', '**/node_modules/**/*'],
+    exclude: ['**/node_modules/**/*'],
```

Comment:
we don't need this at all now. node_modules are already excluded without specifying them, only added here as overriding this config